pkg/cosign/verify.go

```diff
@@ -674,6 +683,15 @@ func verifyInternal(ctx context.Context, sig oci.Signature, h v1.Hash,
 			return bundleVerified, err
 		}
 	}
+	// TODO: Verify TSA only and return the result with the tsa bundle
```

Comment:
Like what I mentioned above, the timestamp augments the Rekor bundle rather than supplants it. It's a way to fetch a timestamp from another source. I'd like the verification flow to look more like what's been done in verifyblob, but we should do that refactor outside of this PR imo. For now, I'd propose something like:
* Add in TS verification and check certificate expiration in VerifyTSABundle
* If verification fails, exit
* If succeeds, pass a flag to `VerifyBundle` that says "don't check the timestamp" ([here](https://cs.github.com/sigstore/cosign/blob/44a7117e0cee495a163c92e9f347efb2d9b747c7/pkg/cosign/verify.go?q=verifybundle+org%3Asigstore#L907)) since it was already checked during TS verification. And also [here](https://cs.github.com/sigstore/cosign/blob/44a7117e0cee495a163c92e9f347efb2d9b747c7/pkg/cosign/verify.go?q=tlogValidateCertificate+org%3Asigstore#L462) when calling Rekor online, we don't need to check expiration if the TS is valid and expiration passes

I think we should still check Rekor/bundle if a timestamp is provided, unless we add flags to control this behavior to skip a Rekor check.

cc @asraa

Reply:
+1 I think if we should still verify the Rekor bundle. Feel free to followup with a flag to control this behavior (--skip-tlog-verify)

Reply:
Yes, blob support will be done as part of a different PR.

I'll add that `skip-tlog-verify` flag as part of a follow-up PR.